.gitignore

```diff
@@ -1,7 +1,6 @@
 # Local .terraform directories
 **/.terraform/*
 tests/*
-*.sh
```

Comment:
Current scripts were already excluded as they all live in `output/` or `.terraform/`, so it's safe to remove this so that modules can have `.sh` templates.